src/app/globals.css

```
@@ -1,35 +1,61 @@
 @import "tailwindcss";
+
 @plugin "daisyui" {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)